Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,63 @@
 ﻿using System;
+using static System.Math;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        /*  Homework #1 
+            Read name, surename, age, weight (in kg) and height (in cm) from console.
+            1) Print all the info based on the example message below:
+                Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm. 
+            2) Calculate and print body-mass index (BMI)
+            3) Do 1 and 2 for another person.
+        */
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string decision;
+            do
+            {
+                Console.Clear(); //Clear console window
+                decision = Add_new_person();
+
+            } while (decision == "Y"); //Repeat if user wants to add new person's data
+        }
+
+        private static string Add_new_person()
```

Comment:
You might want to take a look to c# naming conventions: https://www.dofactory.com/reference/csharp-coding-standards